.github/workflows/cli.yml

```diff
@@ -58,37 +58,35 @@ jobs:
       - name: ⬇️ Fetch commits from base branch
         run: git fetch origin ${{ github.event.before || github.base_ref || 'main' }}:${{ github.event.before || github.base_ref || 'main' }} --depth 100
         if: github.event_name == 'pull_request' || github.event_name == 'workflow_dispatch'
+
+      - name: ♻️ Restore Yarn Cache
```

Comment:
We already have an internal caching action, would be nice to reuse its logic instead, and support caches we are missing within it's code:
* https://github.com/expo/expo/blob/8dd0f0277dc657f0eedc2a11c484369ad135788b/.github/actions/expo-caches/action.yml#L1-L1

Reply:
Removing that is primarily what's speeding up the CI tests.

Reply:
That's interesting! Let's circle back on this separately with infra team.